packages/vest/src/core/state/symbols.js

```diff
@@ -0,0 +1,3 @@
+export const SYMBOL_STATE = Symbol('vestState');
```

Comment:
What is the point of the string sent to the Symbol? I know that Symbol() !== Symbol() anyway. Are you using this string in some way or you're using the variables as unique values? (The second means that you may don't need the string values.

Reply:
I use the descriptor for easier debugging.